### ISSUE TITLE
Error while installing using php7.1rc1

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/core#26122]
### Steps to reproduce:

1. Install oC.

### Expected behaviour:
It is installed as usually.

### Actual behaviour
Error while creating admin user.

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
7.1 RC1

**ownCloud version:** (see ownCloud admin page)
Current master
{"installed":true,"maintenance":false,"version":"9.2.0.1","versionstring":"9.2.0 pre alpha","edition":"","productname":"ownCloud"}

**Updated from an older ownCloud or fresh install:**
Fresh

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
No

**Logs** 

```
{"reqId":"zlQJ7JFbQyluLB7WCZgH","remoteAddr":"::1","app":"index","message":"Exception: {\"Exception\":\"Error\",\"Message\":\"[] operator not supported for strings\",\"Code\":0,\"Trace\":\"#0 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/App\\\/InfoParser.php(188): OC\\\\App\\\\InfoParser->xmlToArray(Object(SimpleXMLElement))\\n#1 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/App\\\/InfoParser.php(59): OC\\\\App\\\\InfoParser->xmlToArray(Object(SimpleXMLElement))\\n#2 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/app.php(663): OC\\\\App\\\\InfoParser->parse('\\\/opt\\\/owncloud\\\/a...')\\n#3 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Installer.php(527): OC_App::getAppInfo('user_ldap')\\n#4 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Setup.php(379): OC\\\\Installer::installShippedApps()\\n#5 \\\/opt\\\/owncloud\\\/core\\\/Controller\\\/SetupController.php(64): OC\\\\Setup->install(Array)\\n#6 \\\/opt\\\/owncloud\\\/lib\\\/base.php(834): OC\\\\Core\\\\Controller\\\\SetupController->run(Array)\\n#7 \\\/opt\\\/owncloud\\\/index.php(39): OC::handleRequest()\\n#8 {main}\",\"File\":\"\\\/opt\\\/owncloud\\\/lib\\\/private\\\/App\\\/InfoParser.php\",\"Line\":186}","level":3,"time":"2016-09-16T07:39:01+00:00","method":"GET","url":"\/index.php","user":"--"}

```

**Comment by SergioBertolinSG** [comment]
@PVince81 it is just a php RC1 version, if you think this is not a bug please close it.

**Comment by PVince81** [comment]
A RC1 isn't supposed to have many bugs.
I think we'll have to adjust our code eventually to match PHP 7's stricter requirements.

**Comment by PVince81** [comment]
```
  "message": "Exception: {"Exception":"Error","Message":"[] operator not supported for strings","Code":0,"Trace":"
#0 \/opt\/owncloud\/lib\/private\/App\/InfoParser.php(188): OC\App\InfoParser->xmlToArray(Object(SimpleXMLElement))\n
#1 \/opt\/owncloud\/lib\/private\/App\/InfoParser.php(59): OC\App\InfoParser->xmlToArray(Object(SimpleXMLElement))\n
#2 \/opt\/owncloud\/lib\/private\/legacy\/app.php(663): OC\App\InfoParser->parse('\/opt\/owncloud\/a...')\n
#3 \/opt\/owncloud\/lib\/private\/Installer.php(527): OC_App::getAppInfo('user_ldap')\n
#4 \/opt\/owncloud\/lib\/private\/Setup.php(379): OC\Installer::installShippedApps()\n
#5 \/opt\/owncloud\/core\/Controller\/SetupController.php(64): OC\Setup->install(Array)\n
#6 \/opt\/owncloud\/lib\/base.php(834): OC\Core\Controller\SetupController->run(Array)\n
#7 \/opt\/owncloud\/index.php(39): OC::handleRequest()\n
#8 {main}","File":"\/opt\/owncloud\/lib\/private\/App\/InfoParser.php","Line":186}",
```

**Comment by PVince81** [comment]
Hmmm no idea... maybe `$array` became a string here for a reason, maybe libxml works differently

**Comment by DeepDiver1975** [comment]
There is a pr open to add 7.1 support

**Comment by DeepDiver1975** [comment]
Wip

**Comment by RealRancor** [comment]
Ref: https://github.com/owncloud/core/pull/25346

**Comment by PVince81** [comment]
@SergioBertolinSG is this still an issue with latest master ? We merged 7.1 support recently.

**Comment by SergioBertolinSG** [comment]
It is working fine now.
